sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 ### Features Added
 
 ### Breaking Changes
+- When calling `BlobContainerClient.getBlobClient(String blobName)` or `BlobClient.getBlobName()` with a URL-encoded 
+blob name, the blob name will no longer be decoded. This is to ensure that the blob name is not modified in any way. If 
```

Comment:
I think this changlog entry is ambiguous and could confuse customers.  We should explain the new logic the way I did in my email about this feature.